File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        StringBuilder answer = new StringBuilder();
        
        for (int i = 0; i < my_string.length(); i++) {
            char c = my_string.charAt(i); 
            if (Character.isLowerCase(c)) { 
                answer.append(Character.toUpperCase(c)); 
            } else {
                answer.append(Character.toLowerCase(c)); 
            }
        }
        
        return answer.toString(); 
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.kt
class Solution {
    fun solution(my_string: String): String {
        var answer: String = ""
        for (i in my_string){
            if (i.isLowerCase()){
                answer += i.toUpperCase()
            }else {
                answer += i.toLowerCase()
            }
        }
        return answer
    }
}